src/main/java/com/parth/jwt/config/JwtHelper.java

```diff
@@ -0,0 +1,69 @@
+package com.parth.jwt.config;
```

Comment:
Why do all 3 of you have the same code?
We already talked about having two apps: the Identity Provider (IDP) App and the Student App. But despite that, all three of you still made the same mistake and added the code into just one app.